components/financial_assistance/app/controllers/financial_assistance/relationships_controller.rb

```diff
@@ -4,8 +4,9 @@ module FinancialAssistance
   class RelationshipsController < FinancialAssistance::ApplicationController
     before_action :find_application
     before_action :set_cache_headers, only: [:index]
+    before_action :enable_bs4_layout, only: [:index]
 
-    layout 'financial_assistance_nav'
+    layout 'financial_assistance_progress'
```

Comment:
this should be feature flagged

Reply:
good catch, https://github.com/ideacrew/enroll/pull/3942/commits/2512012ee2cf9a4bd58c19a5b4148815bb2f439e